autoload/tagbar/prototypes/basetag.vim

```diff
@@ -85,9 +79,9 @@ function! s:_getPrefix() abort dict
     " Visibility is called 'access' in the ctags output
     if g:tagbar_show_visibility
         if has_key(self.fields, 'access')
-            let prefix .= get(s:visibility_symbols, self.fields.access, ' ')
+            let prefix .= get(g:tagbar_visibility_symbols, self.fields.access, ' ')
```

Comment:
This doesn't look like it will work quite right to me. Setting custom ones as in your use case will work because `g:tagbar_visibility_symbols` will be defined, but in the case it is not, the default values entered here will nod be found because they are in in a different array. This needs to fallback to the default if an override is not found.